content/en/blog/_posts/storage in k8s /index.md

```diff
@@ -0,0 +1,170 @@
+
+For this storage requirements looks like :
+
+
+
+* Storage does not depends on the Pod LifeCycle
+* Storage must be available on all node
+* Storage needs to survive even if cluster crashes 
+
+
+![Alt text](image-1.png)
+
+In favor to create  the persistent volume, it create just like any other component get create using yaml
+
+
+
+* Create as cluster resource
+* Created via YAML file
+    * Kind: persistent volume
+    * Spec: How much storage is required
+
+Persistent volumes need actual storage, such as local disks, external interface servers, or cloud storage. Now the question is, Where does this storage backend come from, and how do you make it available to the cluster? 
+
+Kubernetes doesn't care about the actual storage, it gives you a PersistentVolume as an interface to the actual storage. As a maintainer or administrator, you have to take care and need to decide what type of storage you need: local or remote. It's important to think of storage in Kubernetes as an external plugin to your cluster whether is a local storage or cloud storage.
+
+
+## **Persistent Volume YAML Example**
+
+By creating Persistent Volumes (PVs), you can utilize actual physical storage. In the PV specification section, you define the storage backend you want to use to create that storage abstraction or resource for applications.
+
+Persistent volumes are the resources and they need to be there before the pod is created. In order to make it available you need to follow steps
+
+
+
+* Configure storage in cluster
+* Create PV component from storage backend 
+* Create explicit configure YAML to use those PV component or claim that volume using PersistentVolumeClaim (PVC)
+
+Here's an example where we use an NFS storage backend. Essentially, specify how much storage is required and provide additional parameters, such as whether it should be read-write or read-only, among other access options.
+
+
+```
+apiVersion: vl
+kind: PersistentVolume
+metadata:
+  name: pv-name
+spec:
+  capacity:
+    storage: 10Gi  //Define how much storage you require
+    VolumeMode: Filesystem 
+    accessModes:
+      - ReadWriteOnce //Define the access mode
+    persistentVolumeReclaimPolicy: Recycle 
+    storageClassName: slow 
+    mountOptions:
+      - hard
+      - nfsvers=4.0
+    Nfs: //Define NFS parameters
+      path: /dir/path/on/nfs/server 
+      server: nfs-server-ip-address
+
+```
+
+
+_Note : Depending on the storage type in the backend, some attributes in the specification will vary because they are specific to the storage details. In the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/volumes/#volume-types), you can find a comprehensive list of over 25 storage backends that Kubernetes supports._
```

Comment:
We use site-relative links where feasible.

```suggestion
_Note : Depending on the storage type in the backend, some attributes in the specification will vary because they are specific to the storage details. In the official [Kubernetes documentation](/docs/concepts/storage/volumes/#volume-types), you can find a comprehensive list of over 25 storage backends that Kubernetes supports._
```